### PR TITLE
Added lint and isort tasks to Pipfile.

### PR DESCRIPTION
**Pull request by jerbob**
I've removed the `runflake8` and `runisort` files, and replaced them with pipenv scripts.

### CODE REVIEW

**Comment by okraits** [comment]
Looks good to me :+1: 

Regarding `runtests.py`, it's a little bit unfortunate that it's not a one-liner script. This is also the case for some other OpenWISP modules. So these files can't simply be replaced completely with pipenv scripts.

Or do you have an idea how to solve this?

**Comment by jerbob** [comment]
The best I can do at the moment is add a pipenv task for `python runtests.py`, that way we'll be able to use `pipenv run test foo bar` in the same way.

**Comment by jerbob** [comment]
A possible solution to the problem could be to use [pytest-django](https://pytest-django.readthedocs.io/en/latest/) to handle unit tests, but that seems out of scope for this PR.

File: Pipfile [hunk 1]
```diff
@@ -12,3 +12,8 @@ coverage = "*"
 coveralls = "*"
 isort = "*"
 flake8 = "*"
+
+[scripts]
+lint = "python -m flake8 --max-line-length=110 --exclude=migrations,./tests/*settings*.py"
+isort = "python -m isort --check-only --recursive --diff"
```
Comment (nemesifier) [inline Pipfile]: can we move this logic to the [setup.cfg as we have done in django-freeradius](https://github.com/openwisp/django-freeradius/blob/master/setup.cfg)? That way the flake8 and isort default options are used also without pipenv.

**Review by nemesifier — COMMENTED** [review]
See my comment below.

Yes using pytest is out of scope here and we don't have an urgent need to do that.

I have not seen any changes to the `.travis.yml` here, I think we should work in a way so that we maintain feedback with the travis-ci build, in order to do that it would be better if @AnonGuy just sent this pull request to the main repository of openwisp-controller so a travis build would be automatically initiated.

What do you think guys?

File: Pipfile (hunk 1 above)
Comment (jerbob) [reply Pipfile]: Yeah, that's a good idea.

Comment (jerbob) [reply Pipfile]: I've addressed this in the most recent commit.

**Comment by jerbob** [comment]
That sounds good to me, it'd be much easier to test the Travis configuration in the upstream repo.

**Comment by okraits** [comment]
@AnonGuy I agree. I suggest that you could start to improve `.travis.yml` based on my changes to it.

**Comment by nemesifier** [comment]
you can close this guys :-) thx

**Comment by okraits** [comment]
@AnonGuy Thanks again for your work!

**Pull request closed without merging**